release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -476,6 +476,11 @@ In {product-title} {product-version} you can now configure the Ingress Controlle
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-controller-configuration-parameters_configuring-ingress[Ingress controller configuration parameters].
 
+[id="ocp-4-9-networking-network-policies-in-console"]
+==== Create network policies in the web console
+
+Logging in to the web console with the `cluster-admin` role now enables you to create new network policies in any namespace in the cluster from a form in the console as well as directly in YAML.
```

Comment:
IBM Style Guide says not to use _as well as_ to mean _and_. Does that apply here? 
Although, this doesn't seem to be a rule we follow very closely.